tests/Makefile

```diff
@@ -213,6 +213,7 @@ $(TOP)/tools/common/SdkVersions.cs: $(TOP)/tools/common/SdkVersions.cs.in
 
 xharness/xharness.exe: $(xharness_dependencies) test.config test-system.config .stamp-src-project-files $(TOP)/tools/common/SdkVersions.cs
 	$(Q_GEN) $(SYSTEM_MSBUILD) /restore $(MSBUILD_VERBOSITY_QUIET) xharness/xharness.csproj
+xharness/xharness.csproj.inc: export BUILD_VERBOSITY=$(XBUILD_VERBOSITY)
 xharness/xharness.csproj.inc: export ABSOLUTE_PATHS=1
```

Comment:
Looks like rule redefinition here. Doesn't GNU make ignore the first one?

Reply:
This isn't actually a rule definition, it's a [target specific variable declaration](https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html).

It's setting the `BUILD_VERBOSITY` variable for _only_ the `xharness/xharness.csproj.inc` target.

Reply:
Thanks for explaining, learning something new every day...

Reply:
ditto, and I though I new make.